images/salt-master/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos:7.6.1810
+FROM base-centos-image
```

Comment:
Pass this in as an `ARG`. That way, we can have the 'base image' name based on a name/tag that changes when the base image changes. For tracking purposes.

Reply:
Better even, retrieve the hash of the base image and use that in FROM.